lib/logger/logger.test.js

```diff
@@ -0,0 +1,49 @@
+const Logger = require(`./logger`);
+const mockConsole = {
+  log: jest.fn(),
+  error: jest.fn()
+};
+global.console = mockConsole;
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+describe(`Should only print for correct levels`, () => {
+  test(`for debug`, () => {
+    testLogging("DEBUG");
+  });
+  test(`for info`, () => {
+    testLogging("INFO");
+  });
+  test(`for warning`, () => {
+    testLogging("WARNING");
+  });
+  test(`for error`, () => {
```

Comment:
Why are you not calling `testLogging` instead in this test?

Reply:
Since `testLogging` only checks `console.log` and error calls `console.error`

Reply:
Ah!